tests/question_form.js

```diff
@@ -1,135 +1,176 @@
-import 'jsdom-global/register';
-import React from 'react'; // eslint-disable-line no-unused-vars
-import {mount} from 'enzyme';
-import {expect} from 'chai';
-import QuestionForm from '../census/ui_app/QuestionForm';
-
-let qsSchema = [
-  {
-    defaultProperties: {
-      enabled: true,
-      required: true,
-      visible: true
+'use strict';
+
+/* eslint-disable max-len */
+
+require('jsdom-global')();
+const _ = require('lodash');
+const React = require('react'); // eslint-disable-line no-unused-vars
+const mount = require('enzyme').mount;
+const expect = require('chai').expect;
+const QuestionForm = require('../census/ui_app/QuestionForm');
+
+describe('<QuestionForm />', () => {
+  describe('QuestionForm renders each question types', function() {
+    beforeEach(function () {
+      this.baseQSSchema = [
+        {
+          defaultProperties: {
+            enabled: true,
+            required: true,
+            visible: true
+          },
+          id: 'yesno_question',
+          position: 1
+        }
+      ];
+      this.baseQuestions = [
+        {
+          id: 'yesno_question',
+          text: 'Is this a yesno question?',
+          type: 'yesno'
+        }
+      ];
+    });
+
+    it('renders QuestionFieldYesNo type', function() {
+      this.wrapper =
+        mount(<QuestionForm questions={this.baseQuestions} qsSchema={this.baseQSSchema} />);
+      expect(this.wrapper.find('QuestionFieldYesNo')).to.have.length(1);
+    });
+    it('renders QuestionFieldText type', function() {
+      // Replace `type` in baseQuestions.
+      this.baseQuestions[0] = _.assign(this.baseQuestions[0], {type: 'text'});
+      this.wrapper =
+        mount(<QuestionForm questions={this.baseQuestions} qsSchema={this.baseQSSchema} />);
+      expect(this.wrapper.find('QuestionFieldText')).to.have.length(1);
+    });
+  });
+
+  let qsSchema = [
```

Comment:
This is kinda big, if it's shared between tests you should probably get it out of here and add it in via `require()`. Or you may do it anyway, depends on how annoying it is to have it in the test's code.